crates/biome_service/src/settings.rs

```diff
@@ -1443,6 +1446,10 @@ impl OverrideSettingPattern {
         if let Some(expand_lists) = json_formatter.expand {
```

Comment:
```suggestion
        if let Some(expand_lists) = json_formatter.expand.or(formatter.expand) {
```

Reply:
Hey, thanks for the review. I'll fix this.

I'm curious why trailing_commas option above doesn't have that .or() call?

```rs
if let Some(trailing_commas) = json_formatter.trailing_commas {
    options.set_trailing_commas(trailing_commas);
}
```

Reply:
@marko-hologram That's because we don't have the `trailingComma` option in the global `formatter` section: https://biomejs.dev/reference/configuration/#formatter. The option is only available in JS or JSON, and it has the different possible values between JS and JSON.